lib/ruby_lsp/ruby_lsp_rails/runner_client.rb

```diff
@@ -48,7 +48,14 @@ def initialize
           # If we can't set the session ID, continue
         end
 
-        stdin, stdout, stderr, wait_thread = Open3.popen3(
+        env = {}
+        if ENV["BUNDLE_GEMFILE"]
+          env = { "BUNDLE_GEMFILE" => ENV["BUNDLE_GEMFILE"] }
+        end
+
+        # TODO: fix signature
+        stdin, stdout, stderr, wait_thread = T.unsafe(Open3).popen3(
+          env,
```

Comment:
I wonder if we can't simply pass `ENV` here and have it inherit the entire environment.

That would also make it easier to test, you could just add a bogus ENV value and verify that `popen3` gets invoked with it.